client/src/loaders/JsonLoader.js

```diff
@@ -13,12 +13,21 @@ class JsonLoader extends Component {
     this.fetchJson = this.fetchJson.bind(this);
   }
 
+  //TODO: JSON is sent and recieved now. Not sure what happens after that
```

Comment:
Do you want to talk this out or is this leftover from earlier?

Reply:
was leftover stuff! deleted